src/functions_framework/_function_registry.py

```diff
@@ -26,7 +26,7 @@
 
 FUNCTION_SIGNATURE_TYPE = "FUNCTION_SIGNATURE_TYPE"
 HTTP_SIGNATURE_TYPE = "http"
-CLOUDEVENT_SIGNATURE_TYPE = "cloudevent"
+CLOUD_EVENT_SIGNATURE_TYPE = "cloudevent"
```

Comment:
I'm not sure if we should use `CLOUD_EVENT` here, as we're not separating words (not "cloud event").

https://www.python.org/dev/peps/pep-0008/#constants